src/constants.ts

```diff
@@ -0,0 +1,2 @@
+// Taken from: https://github.com/vercel/next.js/blob/f1dbc9260d48c7995f6c52f8fbcc65f08e627992/packages/next/server/config-shared.ts#L161
+export const DEFAULT_PAGE_EXTENSIONS = ["tsx", "ts", "jsx", "js"];
```

Comment:
Could this instead be imported from `next`? https://github.com/vercel/next.js/blob/f1dbc9260d48c7995f6c52f8fbcc65f08e627992/packages/next/server/config-shared.ts#L144

Reply:
Technically yes, but it would have to be imported via that internal module (`next/dist/server/config-shared`). Next doesn't expose it through the public API.